File: backend/GestaoDeEstoque.Api/Models/Produto.cs
namespace GestaoDeEstoque.Api.Models;

public class Produto
{
    public int Id { get; set; }
    public string Nome { get; set; } = string.Empty;
    public string Descricao { get; set; } = string.Empty;
    public int Quantidade { get; set; }
    public decimal Preco { get; set; }
    
    // ADICIONE ESTAS DUAS PROPRIEDADES QUE FALTAVAM
    public int CategoriaId { get; set; }
    public int FornecedorId { get; set; }
}

File: Sistema-de-gestao-de-estoque/src/models/produto.ts
import { Categoria } from './categoria';
import { Fornecedor } from './fornecedor';

export interface Produto {
  id: number;
  nome: string;
  descricao: string;
  quantidade: number;
  preco: number;
  categoriaId: number; 
  fornecedorId: number;
}